virtio-win-drivers-installer/Drivers/viofs/viofs.wxi

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Include>
+<?define driverName = "viofs" ?>
+<?define srcDirName = "viofs" ?>
+<?define destDirName = "VIOFS" ?>
+<?define with_sys = "true"?>
+<?define extra_components = "true"?>
+<?define permanent = "no"?>
+<?if $(sys.BUILDARCH) = "x64" ?>
+<!-- 2k12 -->
+<?define guid = "0B1F3B06-EB7C-422E-A347-58BE0D1C5E8B"?>
+<?define viofs_WdfCoInstaller_guid = "0CED08C2-7C43-40AA-8453-1D72F4455C19"?>
+<?define viofs_virtiofs_guid = "E27A24F9-EDDA-4F34-ADB6-0E9E8A40AE6A"?>
+<?include ../GenericWindowsComponets/2k12.wxi ?>
+<?undef viofs_WdfCoInstaller_guid?>
+<?undef viofs_virtiofs_guid?>
+<?undef guid?>
+<!-- 2k12R2 -->
+<?define guid = "E790AD8E-A7F8-4182-AC87-4FCA639D125E"?>
+<?define viofs_WdfCoInstaller_guid = "62646C3D-06D7-4E84-989B-F13CF317A341"?>
+<?define viofs_virtiofs_guid = "90139462-F500-4411-A1D2-CCC4093B6A8E"?>
+<?include ../GenericWindowsComponets/2k12R2.wxi ?>
+<?undef viofs_WdfCoInstaller_guid?>
+<?undef viofs_virtiofs_guid?>
+<?undef guid?>
+<!-- 2k16 -->
+<?define guid = "3D1814BB-CDF2-4918-BD34-EA697A14D63B"?>
+<?define viofs_virtiofs_guid = "7F51A186-28F4-4C7C-986B-279AC73F6C9C"?>
+<?include ../GenericWindowsComponets/2k16.wxi ?>
+<?undef viofs_virtiofs_guid?>
+<?undef guid?>
+<!-- 2k19 -->
+<?define guid = "794FE5BB-E999-4257-A4C1-2E4E9DC4A213"?>
+<?define viofs_virtiofs_guid = "1D146EAB-CC13-45E8-A0BC-FA70A2F913AF"?>
```

Comment:
have you tried building it?
Not sure if it will fail on the missing viofs_WdfCoInstaller_guid (even that it don't use it)
if it passes then great :)